dotnet/Razorvine.Pyrolite/Pyrolite/Pickle/UnpickleStack.cs

```diff
@@ -13,12 +13,12 @@ namespace Razorvine.Pickle
 [SuppressMessage("ReSharper", "InconsistentNaming")]
 public class UnpickleStack {
 	public readonly object MARKER;
-    private const int DefaultCapacity = 4;
+    internal const int DefaultCapacity = 4;
 	private object[] _stack;
     private int _count;
 
-	public UnpickleStack() {
-		_stack = new object[DefaultCapacity];
+	public UnpickleStack(int capacity = DefaultCapacity) {
```

Comment:
Is this an ABI breaking change?

If you tried loading a library that was compiled on the old version of this library against this new version, and the caller was calling `new UnpickleStack()` they would get a `MissingMethodException`, right? Is that something we are concerned with?

Reply:
good catch! I added an overload instead.